src/Payum/Stripe/Action/Api/CreateChargeAction.php

```diff
@@ -65,6 +66,14 @@ public function execute($request)
         try {
             Stripe::setApiKey($this->keys->getSecretKey());
 
+            if (method_exists(Stripe::class, "setAppInfo")) {
```

Comment:
The next release of Payum will use version 7 of the Stripe SDK (#839), so this method should always exist then, and this check can be removed.